pandas/core/base.py

```diff
@@ -1141,7 +1142,10 @@ def _map_values(self, mapper, na_action=None):
             values = self._values
             if na_action is not None:
                 raise NotImplementedError
-            map_f = lambda values, f: values.map(f)
+
+            def map_f(values, f):
+                return values.map(f)
+
```

Comment:
this is changed to silence `pandas/core/base.py:1151: note: Internal mypy error checking function redefinition`